packages/admin/admin/package.json

```diff
@@ -46,6 +46,7 @@
         "@babel/cli": "^7.17.6",
         "@babel/core": "^7.20.12",
         "@comet/admin-babel-preset": "workspace:^6.4.0",
+        "@comet/admin-theme": "workspace:^6.4.0",
```

Comment:
If we now need the theme-package inside the admin-package, maybe it no longer makes sense, to keep them as separate packages... what do you think @johnnyomair?

Reply:
Also, I think this is a breaking change, since it is now required to use `@comet/admin-theme` when using `@comet-admin`.

Is there a way we can avoid that? 🤔

Reply:
Another implication of this is:
In `@comet/cms-admin` we have `@comet/admin-theme` as a normal dependency. But now we actually depend on the fact that the application uses `createCometTheme` which would make it a peer dependency.

Reply:
> We are planning to merge all the admin packages anyway. We just didn't have the time yet. Maybe in v8.

👍🏼 

> That is breaking. But I don't think we can avoid that if we want to use our custom variants in `@comet/admin`

Then we should change target branch to `next` and make this part of V7. 
Or is this needed in V6 for some reason?

Reply:
I'd say we target v7 instead. While it probably won't be a problem for any of our projects, it's still a breaking change. Also, the new variants aren't needed in projects yet AFAIK.

Reply:
I proceeded to make admin-theme a peer dependency of admin and cms-admin